.gitignore

```diff
@@ -22,11 +24,25 @@ node_modules/
 
 # Various OS/editor specific files
 *#
+*.orig
+*.rej
+*.svn
 *.sw[po]
+*.xcodeproj
+*Thumbs.db
 *~
-\#*
-scratch
+.DS_Store
+.directory*
 .idea/
 .vscode/
-.DS_Store
-*.rej
+\#*
+scratch
+
+# Testsuite-specific rules
+/conformance-checkers/vnu.jar
+/cors/resources/log.txt
+/css//build-temp
+/css/dist
+/css/dist_last
+/css/tools/cache
+/webaudio/idl/*
```

Comment:
Probably should have trailing new line. (Also, do we actually want `/*` at the end here? That's very surprising.)

Reply:
I literally just copied the existing rule. I'm not sure what the point of it actually is.